app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -82,6 +84,40 @@ class SyncIntegrationTest {
         }
     }
 
+    @Test
+    fun checkLoginsFromDesktopTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+            // Necessary to scroll a little bit for all screen sizes
+            TestHelper.scrollToElementByText("Logins and passwords")
+        }.openLoginsAndPasswordSubMenu {
+        }.openSyncLogins {
+            // Tap to sign in from Logins menu
+            tapOnUseEmailToSignIn()
+            typeEmail()
+            tapOnContinueButton()
+            typePassword()
+            tapOnSignIn()
+        }
+        // Automatically goes back to Logins and passwords view
+        settingsSubMenuLoginsAndPassword {
+            mDevice.waitNotNull(Until.findObjects(By.text("Sync logins")), TestAssetHelper.waitingTime)
+            verifyDefaultView()
+            mDevice.waitNotNull(Until.findObjects(By.text("Off")), TestAssetHelper.waitingTime)
+        }.openSavedLogins {
+            // Discard the secure your device message
+            tapSetupLater()
+            // Check the logins synced
+            mDevice.waitNotNull(Until.findObjects(By.text("https://accounts.google.com")), TestAssetHelper.waitingTime)
+            verifySavedLoginsAfterSync()
+        }.goBack {
+            // After checking the synced logins
+            // on Logins and Passwords menu the Sync logins option is set to On
+            mDevice.waitNotNull(Until.findObjects(By.text("On")), TestAssetHelper.waitingTime)
+        }
+    }
+
     /* These tests will be running in the future
```

Comment:
for this are we waiting on a feature to land?

Reply:
Not really, I just need to work on those one which are in the direction Fenix -> Desktop